CHANGELOG.md

```diff
@@ -0,0 +1,135 @@
+## To be released
```

Comment:
True, but in this case if you try to release it, mobitron will release an empty changelog, which is not great. Anyway, I agree with your reasoning :)